BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -168,13 +173,16 @@ extension AttributedLabel {
         }
 
         private func updateAccessibilityTraits(_ model: AttributedLabel) {
+
```

Comment:
Nit: Could we just move the `.staticText` setting up here?

I think we'd need to re-add `.staticText` for an update from `.updatesFrequently` back to an empty set for example.

Maybe something like:
```swift
var traits = UIAccessibilityTraits(with: [.staticText])

if let modelTraits = model.accessibilityTraits {
    traits = traits.union(.init(with: modelTraits))
    // UILabel has the `.staticText` trait by default. If we explicitly set `.updatesFrequently` this should be removed.
    if traits.contains(.updatesFrequently) {
        traits.subtract(.staticText)
    }
}

accessibilityTraits = traits
```

Reply:
Will do

Reply:
This means that `[]` won't _remove_ `.staticText`, which I guess is fine?

Reply:
Hmm, might be better to respect `[]`? Not strong opinion

Reply:
Sounds good, updating!

Reply:
Actually, hm – it seems like for labels, you'll always want either `.staticText`, or `.updatesFrequently` – if you want to disable the element, you can set `.isAccessibilityElement = false`. So maybe always including it does make sense?

Reply:
Looping in @RoyalPineapple

Reply:
Good summary here: https://mobilea11y.com/blog/traits/

I think we can be a lot smarter in the label itself about this and not require these traits at all. In the mean time I'll get this working how it was before

Reply:
In the mean time, West pointed out that we should just fix the original bug. And not change the behaviour – so doing that.

Reply:
To clarify I do think we should follow up with improving this with the help of @RoyalPineapple! I just want to get a release out with existing behavior if possible and would like to give the better fix enough time.